yb-voyager/cmd/root.go

```diff
@@ -77,7 +76,14 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 			}
 			InitLogging(bulkAssessmentDir, cmd.Use == "status", GetCommandID(cmd))
 			startTime = time.Now()
+			log.Infof("Start time: %s\n", startTime)
+
+			if callhome.SendDiagnostics {
+				createCLIArgsString(cmd)
+				go sendCallhomePayloadAtIntervals()
```

Comment:
There is nothing really that changes frequently in bulk-assessment. Don't think we need this.

Reply:
Right, this goroutine is for long running data commands, so we don't need this bulk command